lib/ansible/module_utils/digital_ocean.py

```diff
@@ -93,8 +93,8 @@ def delete(self, path, data=None):
 
     def get_do_oauth_token(self):
         try:
-            self.oauth_token = self.module.params['oauth_token'] or \
-                self.module.params['api_token'] or \
+            self.oauth_token = self.module.params.get('oauth_token') or \
+                self.module.params.get('api_token') or \
```

Comment:
Should the environment variables also be looked for with get()?

Reply:
That's how AWS credentials are handled too. https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/ec2.py#L187

Besides that, this looks good to me.

Reply:
Thank you I was looking for an example! I figured those would need to be included also but wasn't sure. New commit has been added with a change to the environment variables.